protected/migrations/fix_import/m220218_062834_update_url_image_tab.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class m220218_062834_update_url_image_tab extends CDbMigration
+{
+	public function safeUp()
+	{
+	    Yii::app()->db->createCommand("update image set url = replace(url, 'http://gigadb.org/','https://assets.gigadb-cdn.net/images/datasets/') where url like 'http://gigadb.org/%';")->execute();
+	    Yii::app()->db->createCommand("update image set url = 'https://assets.gigadb-cdn.net/images/datasets/no_image.png' where url like '' and location like 'no_image%';")->execute();
```

Comment:
Do we need an acceptance test for this? Might be useful to create some data for a very basic private dataset that uses the generic no image png file?

Reply:
Agree with @pli888 that a scenario is missing.

It's more about testing the case where a generic image ought to be displayed. 

you want a scenario that uses a dataset whose image record fulfills the condition for generic image to be used. there may be such test data in the `dev` test data already, but if it's not the case, we need to had one.

Reply:
Hi @pli888  and @rija,

A scenario `Dataset with no image associated will show generic image` has been created.
In order to pass it, a dataset `/dataset/300070` is created with empty url  and location with value `no_image.png`. 

A new command `update image set url = '' where url is null;` is created to handle null value which should not be existed during test database set up from csv files.

Reply:
Hi @pli888,

> Rather than having to manually execute the above command to replicate what will soon be the situation on [gigadb.org](http://gigadb.org/) with the images fetched from CDN, would it be better to add the command to the `ops/scripts/setup_*db.sh` scripts?

My idea is that this fix will only be used once or several times during development, but after that, this script will be no longer needed because all data stored would be in a correct format. 
And the setup script is a more permanent way, because it will be called whenever `./up.sh` is called.